sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/finishAnalysis.scala

```diff
@@ -91,6 +94,7 @@ object ComputeCurrentTime extends Rule[LogicalPlan] {
         })
       case CurrentTimestamp() | Now() => currentTime
       case CurrentTimeZone() => timezone
+      case LocalTimestamp() => localTime
```

Comment:
> I don't see much difference

@gengliangwang The difference is in that you don't need to especially handle `LocalTimestamp()` here, for instance, and maybe in other places.